src/lsst/cmservice/handlers/script_handler.py

```diff
@@ -384,6 +359,40 @@ async def _check_slurm_job(  # pylint: disable=unused-argument
             await script.update_values(session, status=status)
         return status
 
+    async def _check_htcondor_job(  # pylint: disable=unused-argument
```

Comment:
This is a generic feature of the _all_ the checking functions, we pass along the parent of the script case we want to inform the parent node about something that we learned as part of the checking.    I don't know that we are using it much, but it seemed useful at some point.

Reply:
On a small scale that's a tradeoff between future utility inside the function vs additional infrastructure required to support those parameters that aren't used. In the big picture, I think parents should be introspecting the status of their constituent Scripts/jobs/etc rather than Scripts modifying their parent.